File: 2017_Summer/Calculator/Calculator.py
while True:
    print("========================================")
    print("계산할 수식(+,-,*,/,%) <Ctrl + C로 중단>")
    
    result = 0.0;

    num1 = int(input("num1:"))
    sel = input("연산자:")
    num2 = int(input("num2:"))

    if(sel == "+"):
        result = (num1 + num2)
    elif(sel == "-"):
        result = (num1 - num2)
    elif(sel == "*"):
        result = (num1 * num2)
    elif(sel == "/"):
        result = (num1 / num2)
    elif(sel == "%"):
        result = (num1 % num2)
    else:
        print("Wrong input, please input again")
        continue

    print("결과: ", result)

File: 2017_Summer/Calculator/Calculator.js
var rs = require("readline-sync");
var num1 = rs.question("num1: ");
var op = rs.question("연산자: ");
var num2 = rs.question("num2: ");
var result;

switch (op) {
    case "+":
        result = parseFloat(num1) + parseFloat(num2);
        break;
    case "-":
        result = parseFloat(num1) - parseFloat(num2);
        break;
    case "*":
        result = parseFloat(num1) * parseFloat(num2);
        break;
    case "/":
        result = parseFloat(num1) / parseFloat(num2);
        break;
}

console.log('결과: ' + result)

if (!num1 || !num2) {
    console.log("Please enter a valid number");
}